projects/canopy/src/styles/typography-fonts.scss

```diff
@@ -1,90 +1,46 @@
-$fonts-path: '../assets/fonts' !default;
-$fonts-path-lyon: '/assets/fonts' !default;
+$fonts-path-otto: '../assets/fonts/otto' !default;
+$fonts-path-nunito-sans: '../assets/fonts/nunito-sans' !default;
 
 /*
-  Lyon - font path is not transformed by the build, see USAGE.md for details
+  Nunito Sans
  */
 @font-face {
-  font-family: 'Lyon';
+  font-family: 'Nunito Sans';
+  font-weight: 100 700;
   font-style: normal;
-  font-weight: 100;
   font-display: swap;
-  src: local('Lyon'), local('Lyon-Regular'),
-    url('#{$fonts-path-lyon}/lyon/lyon-display-regular.woff2') format('woff2');
-}
-
-@font-face {
-  font-family: 'Lyon';
-  font-style: normal;
-  font-weight: 300;
-  font-display: swap;
-  src: local('Lyon'), local('Lyon-Regular'),
-    url('#{$fonts-path-lyon}/lyon/lyon-display-regular.woff2') format('woff2');
-}
-
-@font-face {
-  font-family: 'Lyon';
-  font-style: normal;
-  font-weight: 400;
-  font-display: swap;
-  src: local('Lyon Bold'), local('Lyon-Bold'),
-    url('#{$fonts-path-lyon}/lyon/lyon-display-bold.woff2') format('woff2');
-}
-
-@font-face {
-  font-family: 'Lyon';
-  font-style: normal;
-  font-weight: 500;
-  font-display: swap;
-  src: local('Lyon Bold'), local('Lyon-Bold'),
-    url('#{$fonts-path-lyon}/lyon/lyon-display-bold.woff2') format('woff2');
+  src: url('#{$fonts-path-nunito-sans}/NunitoSans-VariableFont.ttf') format('truetype');
```

Comment:
I wonder if there would be any issue using `ttf` (https://caniuse.com/?search=ttf). I remember we previously had to use `woff2` because it was more supported.
I personally say it's ok and whatever we don't support will fall back to Arial, etc.

@IdaHW and @alexcanning what do we think?

This is the support for most of our apps:
```
last 2 chrome major versions
last 2 edge major versions
Safari >= 15
last 2 firefox major versions
last 2 ChromeAndroid major versions
last 2 iOS major versions
last 2 Samsung major versions
```